libp2p/host/ping.py

```diff
@@ -14,6 +20,21 @@
 logger = logging.getLogger("libp2p.host.ping")
 
 
+async def handle_ping(stream: INetStream) -> None:
+    """``handle_ping`` responds to incoming ping requests until one side errors
+    or closes the ``stream``."""
+    peer_id = stream.muxed_conn.peer_id
+
+    while True:
+        try:
+            should_continue = await _handle_ping(stream, peer_id)
+            if not should_continue:
+                return
+        except Exception:
+            await stream.reset()
+            return
```

Comment:
any particular reason to move this chunk of code up here?

just curious.... at first pass on this review i figured you had changed some of the logic but it seems to be the same(!)

Reply:
Oh I'm sorry, I think this happened because at first I moved it inside `PingService` as a method. But at the end I left it out, and I think I moved it accidentally